ldapadmin/src/deb/resources/etc/georchestra/ldapadmin/css/main.css

```diff
@@ -0,0 +1,123 @@
+#container {
```

Comment:
`css/main.css` and `js/config.js` are customized in the current build process (see my comment below). If we want to stay iso-functional, I have not choice than putting them in the datadir, so that the administrator can customize them.